### PR TITLE
Platform picker high contrast

### LINKED ISSUE #10952: Office UI Fabric site does not work well in High contrast
Switching to high contrast (I used the black one) the site had many accessibility issues, failing MS accessibility standards. Here are some of them I have seen and captured:

1. Pictures on left navigation for iOS and Android do not have enough color contrast:
 
image

2. Active and focus buttons on navigation have background and font color as black:
 
image

3. When moving around to select a component from a menu on the left the focus is invisible:
 
image

Notice how the link at the bottom shows what I am currently selecting but there is no indication visually what I have selected.

4. On the individual component pages they have color contrast issues as well:
Both the active and the focus have font color and background

image

5. The button color is white and the font color is white
 
image

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #10952
- [X] Include a change request file using `$ yarn change`

#### Description of changes

Adds high contrast support to platform picker.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11089)

### CODE REVIEW

File: packages/example-app-base/src/styles/_mixins.scss [hunk 1]
```diff
@@ -34,3 +34,11 @@
     pointer-events: none;
   }
 }
+
+// High contrast mode mixins
```
Comment (ecraig12345) [inline packages/example-app-base/src/styles/_mixins.scss]: Could you add a reference to `~office-ui-fabric-react/lib/common/_highContrast.scss` instead? (that may not be quite the right path formatting)

Comment (jdhuntington) [reply packages/example-app-base/src/styles/_mixins.scss]: That doesn't seem to work. Looks like `lib/common` has an already-javascript version, so can't be imported by the sass loader.

Comment (ecraig12345) [reply packages/example-app-base/src/styles/_mixins.scss]: Ah it's dist. Similar example: `@import '~office-ui-fabric-core/dist/sass/References';`

Comment (ecraig12345) [reply packages/example-app-base/src/styles/_mixins.scss]: Wait sorry that's an import from core...

Comment (ecraig12345) [reply packages/example-app-base/src/styles/_mixins.scss]: Interesting, for OUFR imports I see other places using both src and dist. So...pick one?
```
@import '~office-ui-fabric-react/src/common/common';
@import '~office-ui-fabric-react/dist/sass/References';
```

**Pull request merged**